Modules/Chatroom/chat/Bootstrap/ReadServerConfig.js

```diff
@@ -1,13 +1,19 @@
 var CONST 		= require('../Constants');
 var Container	= require('../AppContainer');
 var Handler		= require('../Handler/FileHandler');
+var dns 		= require('dns');
```

Comment:
This seems not to belong to your pullrequest which is described to just change the position of an icon. Please remove it from this PR or update the description. Thanks a lot!